osx/app_resources/plover_launcher.c

```diff
@@ -0,0 +1,41 @@
+#include <unistd.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <libgen.h>
+
+#define ARGS_LIST_SIZE sizeof(arg_list) / sizeof(char*)
+
+extern char **environ;
+int main(int argc, char *argv[]) {
+
+    // Relative path from this file.
+    char python_path[] = "/../Frameworks/pythonexecutable";
+
+    // Get this file's path.
+    char path[1048];
```

Comment:
```suggestion
    char path[PATH_MAX];
```
N.B.: `limits.h` needs to be included.